### ISSUE TITLE
Error in the convert timezone function in MongoBiQuery.js

### ISSUE THREAD

**Issue by karkill** [cube-js/cube#7152]
**Describe the bug**
Hi @paveltiunov, I found some wrong code in function `convertz` of **MongoBiQuery.js**. I think it must be 
```js
    if (minuteInt !== 0) {
      result = `TIMESTAMPADD(MINUTE, ${minuteInt}, ${result})`;
    }
 ```
 Please fix it and release in next version.
 
https://github.com/cube-js/cube/blob/v0.33.58/packages/cubejs-schema-compiler/src/adapter/MongoBiQuery.js

**Version:**
v0.33.58

**Additional context**
![Wrong code](https://github.com/cube-js/cube/assets/44264041/f84c3261-7582-4e8b-847e-77c9933bb939)

Original Slack thread: https://cube-js.slack.com/archives/C04NYBJP7RQ/p1695194399940109?thread_ts=1695113090.745349&cid=C04NYBJP7RQ

**Comment by igorlukanin** [comment]
@karkill Thanks for hunting this bug! Would you be open to submit a PR with a fix?

**Comment by karkill** [comment]
Thank for your help @igorlukanin but I don't have permission to create new branch. Please help me fix the bug mentioned above and create a PR.

**Comment by joybh98** [comment]
@karkill if you're not working on it, I would love to take a look.

**Comment by joybh98** [comment]
Hi @igorlukanin , i'd like to to work on it, if @karkill is not !

**Comment by ovr** [comment]
Fix released in [v0.34.25](https://github.com/cube-js/cube/releases/tag/v0.34.25)

Thanks
